linux_os/guide/system/accounts/accounts-session/accounts_users_netrc_file_permissions/oval/shared.xml

```diff
@@ -0,0 +1,49 @@
+<def-group>
+  <definition class="compliance" id="{{{ rule_id }}}" version="1">
+    {{{ oval_metadata("Netrc User File In The Home Directory Must Not be group or world
+    accessible") }}}
+    <criteria>
+      <criterion test_ref="test_accounts_users_home_netrc_file_permissions"
+                 comment="Netrc User File In The Home Directory Must Not be group or world
+                          accessible"/>
+    </criteria>
+  </definition>
+
+  <unix:password_object id="object_accounts_users_home_netrc_file_permissions_objects" version="1">
+    <unix:username datatype="string" operation="pattern match">.*</unix:username>
+    <filter action="include">state_{{{ rule_id }}}_interactive_uids</filter>
+  </unix:password_object>
+
+  <unix:password_state id="state_{{{ rule_id }}}_interactive_uids"
+                       version="1">
+    <unix:user_id datatype="int" operation="greater than or equal">{{{ uid_min }}}</unix:user_id>
+  </unix:password_state>
+
+  <local_variable id="var_accounts_users_home_netrc_file_permissions_dirs" datatype="string"
+                  version="1" comment="Variable including all home dirs from interactive users">
+    <object_component item_field="home_dir"
+                      object_ref="object_accounts_users_home_netrc_file_permissions_objects"/>
+  </local_variable>
+
+  <unix:file_object id="object_accounts_users_home_netrc_file_permissions" version="1">
+    <unix:path var_ref="var_accounts_users_home_netrc_file_permissions_dirs"
+               var_check="at least one"/>
+    <unix:filename operation="pattern match">\.netrc</unix:filename>
+    <filter action="exclude">state_accounts_users_home_netrc_file_permissions</filter>
+  </unix:file_object>
+
+  <unix:file_state id="state_accounts_users_home_netrc_file_permissions" version="1">
+    <unix:gread datatype="boolean">false</unix:gread>
+    <unix:gwrite datatype="boolean">false</unix:gwrite>
+    <unix:gexec datatype="boolean">false</unix:gexec>
+    <unix:oread datatype="boolean">false</unix:oread>
+    <unix:owrite datatype="boolean">false</unix:owrite>
+    <unix:oexec datatype="boolean">false</unix:oexec>
+  </unix:file_state>
+
+  <unix:file_test id="test_accounts_users_home_netrc_file_permissions" check="all"
+                  check_existence="any_exist" version="1"
+                  comment=".netrc files are not group or world accessible">
+    <unix:object object_ref="object_accounts_users_home_netrc_file_permissions"/>
```

Comment:
Could you use the `state_accounts_users_home_netrc_file_permissions` state here, in this test, instead of as a filter in the object, please? It will make even the reports better.